src/platforms/salesforce/constants.ts

```diff
@@ -1,2 +1,9 @@
 export const MAX_QUERY_PAGE_SIZE = 200;
 export const SALESFORCE_API_VERSION = 'v53.0';
+export const SALESFORCE_SUPPORTED_OBJECT_TYPE = [
+  'User',
+  'Contact',
+  'ListView',
```

Comment:
did we need to add support for lists to this PR? https://github.com/vesselapi/core/pull/143/files

Reply:
Lists were here previously from Marketing Automation, I didn't include it in the existing PR though, but I will in a follow-up